dynawo/3rdParty/libzip/include/libzip/ZipEntry.h

```diff
@@ -12,21 +12,27 @@
 #ifndef __ZIP_ENTRY_H__
 #define __ZIP_ENTRY_H__
 
+#include <archive_entry.h>
```

Comment:
Can't we split this huge MR into 1 MR per library? This woul be to limit the risk of a massive breakage